modules/custom-cert-new-install-operator.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * configuration/add-custom-certificates.adoc
+:_module-type: CONCEPT
+[id="custom-cert-new-install-operator_{context}"]
+= Adding a custom certificate during a new installation with Operator
+
+By default, Central only serves an internal TLS certificate, which means you need to handle TLS termination at the ingress or load balancer level.
```

Comment:
This isn't technically correct. It can be passthrough which would serve the internal certificate that we serve by default. It would be correct if you simply removed ", which means you need to handle TLS termination at the ingress or load balancer level."